v2/src/modules/txs/ModuleTxDetails.vue

```diff
@@ -1,56 +1,201 @@
 <template>
-    <!--
-    =====================================================================================
-      TX DETAILS LIST
-    =====================================================================================
-    -->
-    <v-row justify="start">
-        <v-col xs="12">
-            <app-details-list has-title title="Transaction Details" :details="txDetails" :is-loading="isLoading" :max-items="7">
-                <template v-if="!isLoading" #title>
-                    <tx-details-title :status="titleStatus" />
+    <template v-if="!state.hasError">
+        <v-card variant="elevated" elevation="1" rounded="xl" class="pa-4 pa-sm-6">
+            <div class="d-flex">
+                <h2 class="text-h6 font-weight-bold mr-10">Transaction Summary</h2>
+                <template v-if="loadingTransactionHash">
+                    <div class="skeleton-box rounded-xl" style="height: 24px; width: 100px"></div>
                 </template>
-            </app-details-list>
-        </v-col>
-    </v-row>
+                <app-chip v-else :bg="titleBg" rounded="xl">
+                    <div class="d-flex align-center">
+                        <p>{{ titleStatus }}</p>
+                        <v-progress-circular v-if="txStatus === TxStatus.pending" indeterminate color="white" size="15" width="3" class="ml-2" />
+                    </div>
+                </app-chip>
+            </div>
+            <v-row class="mt-5">
+                <v-col cols="12" lg="6">
+                    <div class="tx-info">
+                        <p class="text-button mb-1">Tx Hash</p>
+                        <template v-if="loadingTransactionHash">
```

Comment:
i think its' better to move this guy under the tx hash  ( so only the timestamp is being loaded)